### PR TITLE
Tweak Travis Configuration

### PR DESCRIPTION
**Pull request by optikfluffel**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
-  - 0.10
-notifications:
-  email:
-    on_success: change
-    on_failure: change
+  - "0.8"
+  - "0.10"
+  - iojs
+sudo: false # use container based environment
+# notifications:
```
Comment (fent) [inline .travis.yml]: why are notifications commented out?

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,10 @@
 language: node_js
 node_js:
-  - 0.8
-  - 0.10
-notifications:
-  email:
-    on_success: change
-    on_failure: change
+  - "0.8"
+  - "0.10"
```
Comment (fent) [inline .travis.yml]: how about "0.12"?

**Comment by optikfluffel** [comment]
@fent I'm not sure why that brakes, do you have any idea?

File: .travis.yml (hunk 1 above)
Comment (optikfluffel) [inline .travis.yml]: Because `change` isn't an email and it shouldn't work? Or am I missing something here?

**Comment by fent** [comment]
`change` is a valid value for `on_success`

http://docs.travis-ci.com/user/notifications/

**Comment by fent** [comment]
that happened to me earlier too and I reran the test fine. it works locally too. I'll merge this later.

**Comment by fent** [comment]
Thanks!

**Pull request merged**